freddie/viewsets/generics.py

```diff
@@ -204,6 +204,7 @@ def api_actions(self) -> None:
 
         pk_type = self.pk_type
         status_code = int(HTTPStatus.OK)
+        response_model = getattr(self, 'item_schema', self.schema)  # type: ignore
```

Comment:
for detail view lets use self.schema, this property is changeable in ViewSets

Reply:
oh..but schema is used for db and we couldn't change it for response. Lets stay it, but rename to 'detail_schema'

Reply:
added a test that checks that the scheme will match detail or list schema